src/main/wrapper/ExecutionService.ts

```diff
@@ -153,6 +153,7 @@ export class ExecutionService {
                 logger.info("Exit code received from AST-CLI: " + code);
                 if(code==1){
                     stderr = stdout
+                    reject(stderr)
```

Comment:
make sure that the rest of the codebase is updated to handle these changes appropriately.

Reply:
yes it's ok